mf-polis/common/services.json

```diff
@@ -0,0 +1,29 @@
+[
+   {
+      "img":"//polis.digital-enabler.eng.it/suite/static/imgs/icons/icon_",
```

Comment:
I don't know if you created a new imgs repo, but this URL is to old Suite where we have the imgs for the tools menu that for now we not show, es.: "https://demo.digital-enabler.eng.it/suite/static/imgs/icons/icon_data_visualization.png"
If you use the old one repo you have to leave that URL as was previous.